private/buf/cmd/buf/buf_test.go

```diff
@@ -2419,7 +2419,19 @@ func testRunStdout(t *testing.T, stdin io.Reader, expectedExitCode int, expected
 	)
 }
 
-func testRunStdoutStderr(t *testing.T, stdin io.Reader, expectedExitCode int, expectedStdout string, expectedStderr string, args ...string) {
+func testRunStdoutStderr(t *testing.T, stdin io.Reader, expectedExitCode int, expectedStderr string, args ...string) {
+	appcmdtesting.RunCommandExitCodeStderr(
+		t,
+		func(use string) *appcmd.Command { return NewRootCommand(use) },
+		expectedExitCode,
+		expectedStderr,
+		internaltesting.NewEnvFunc(t),
+		stdin,
+		args...,
+	)
+}
+
+func testRunStdoutStderrNoWarn(t *testing.T, stdin io.Reader, expectedExitCode int, expectedStdout string, expectedStderr string, args ...string) {
```

Comment:
This func doesn't output warnings. It sends `--no-warn` flag.

Reply:
when did we add a `--no-warn` flag? is this part of the buf CLI?

Reply:
oh got it, ok